Assets/Scripts/PlayerMovementTime.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovementTime : MonoBehaviour
+{
+    // Player's rigidbody
+    Rigidbody rb;
+
+    // Player speed
+    [SerializeField] float speed = 3f;
```

Comment:
I believe if we just make these public its by default serialized. See https://docs.unity3d.com/ScriptReference/SerializeField.html

Reply:
Same for other code

Reply:
fixed